cht-petals/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+export VERSION=1.0.4
+source "$(dirname "${BASH_SOURCE[0]}")/../utils.sh"
+
+build_cpu ghcr.io/premai-io/chat-codellama-34b-cpu   codellama-34b   ${@:1}
+build_cpu ghcr.io/premai-io/chat-stable-beluga-2-cpu stable-beluga-2 ${@:1}
```

Comment:
meta-question:

is "the Petals way to use your local GPU" to separately connect it to Petals swarm?
i.e. there are no advantages to `build_gpu`? there's no way to force using both the local GPU and the swarm (without connecting local GPU to the swarm)?

Reply:
No advantages to build_gpu as of now. It can be an improvement that we do. I ll note it down.